pkg/apis/kubeflow/v2beta1/types.go

```diff
@@ -131,6 +139,16 @@ type RunPolicy struct {
 	// Defaults to false.
 	// +kubebuilder:default:=false
 	Suspend *bool `json:"suspend,omitempty"`
+
+	// ManagedBy is used to indicate the controller or entity that manages a job.
```

Comment:
```suggestion
	// ManagedBy is used to indicate the controller or entity that manages a MPIJob.
```

Let's identify which Job is delegated to the external controller.
Because the mpi-operator has batch/v1 Job concept.